src/Immediate.Jobs.Dashboard/DashboardClient/src/styles.css

```
@@ -1034,17 +1047,27 @@ tbody tr[data-selected="true"] td:first-child {
 }
 
 .workflow-edge,
+.workflow-fork path,
 .workflow-join path {
 	fill: none;
 	stroke: var(--text-subtle);
 	stroke-width: 1.4;
+	stroke-linecap: round;
+	stroke-linejoin: round;
 	opacity: 0.75;
+	shape-rendering: geometricPrecision;
```

Stylelint (17.14.0)

[error] Expected "geometricPrecision" to be "geometricprecision" (value-keyword-case)

(value-keyword-case)